pkg/machine/ignition.go

```diff
@@ -203,6 +215,24 @@ func getFiles(usrName string) []File {
 			Mode: intToPtr(420),
 		},
 	})
+
+	// Issue #11489: make sure that we can inject a custom registries.conf
```

Comment:
```suggestion
	// Inject a custom registries.conf that $does_X_because_Y
```
? “can inject/custom”, and “on the system level” seem not to really be the point of this code.

(I don’t feel strongly at all about not including the issue link. A short summary of what/why with a “see link for details/discussion” would be perfectly fine with me; this just triggers, mostly incorrectly, my “don’t use code comments as a change log of fixed bugs” instinct.)